tasks/styles.js

```diff
@@ -45,6 +45,9 @@ module.exports = (gulp, $, pkg) => {
         }),
         outputStyle: options.outputStyle
       }).on('error', reportError))
+      .pipe($.if(options.production, $.cssUrlCustomHash({
```

Comment:
In my opinion the hashes should be enabled at all times, because caching related HTTP response headers should also be enabled at all times, including in development, staging & QA site instances, so that a potential misbehavior is visible as soon as possible.

```suggestion
      .pipe($.cssUrlCustomHash({
```

Reply:
Ok. Let's do it like this 👍